kubernetes/docker/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# This script can be used to build the custom docker images required
+# for Kubernetes. This involves running the entire OpenWhisk gradle
+# build process and then creating the custom images for OpenWhisk.
+
+# prerequisites:
+#   * be able to run `cd <home_openwhisk> ./gradlew distDocker`
+
+set -ex
+
+
+if [ -z "$1" ]; then
+cat <<- EndOfMessage
+  First argument should be location of which docker repo to push all
+  of the built OpenWhisk docker images. This way, Kubernetes can pull
+  any images it needs to.
+EndOfMessage
+
+exit 1
+fi
+
+OPENWHISK_DIR=""
+if [ -z "$2" ]; then
+cat <<- EndOfMessage
+  Second argument should be the location of where the OpenWhisk repo lives.
+  By default the location is $HOME/workspace/openwhisk
+EndOfMessage
+
+  OPENWHISK_DIR=$HOME/workspace/openwhisk
+else
+  OPENWHISK_DIR="$2"
+fi
+
+pushd $OPENWHISK_DIR
+  ./gradlew distDocker
+popd
+
+## Retag new images for public repo
+docker tag whisk/badaction "$1"/whisk_badaction
```

Comment:
I'm not sure what the final script will be. Ideally it would be great to have the CI first build the docker images, use the built images in CI, then if everything passes it would publish them. So this script would need to be in 2 parts and we need an official docker repo. For now I'm not to worried about it. Just something to use when building images locally.